app/views/rounds/_show_question.erb

```diff
@@ -17,8 +17,8 @@
               <div class="btn btn-warning">No I haven't</div>
             <% end %>
           </div>
-
         </div>
+
```

Comment:
Just some alignment adjustments, also this file is the one we need, deleted the one from `app/views/instances/_show_question.erb`